api/database/config/config.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config();
```

Comment:
We will need to import `dotenv` only in dev mode because in production we do read environment variables from the system directly.

Reply:
Can you elaborate more on this, please?

Reply:
`dotenv` read variables from the `.env` file but in production, those values are placed directly on the system where production code is going to run in most cases. If we intend to also use `.env` in production then it is fine to leave it the way it is.